envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/chaos/storage/NoopChaosDataStore.kt

```diff
@@ -2,4 +2,5 @@ package pl.allegro.tech.servicemesh.envoycontrol.chaos.storage
 
 class NoopChaosDataStore : ChaosDataStore {
```

Comment:
Wouldn't it be easier for you to test if you had an in-memory data store for tests instead of a noop implementation?

Reply:
ok, there will be in-memory data store for tests in next PR